rts/Sim/Units/Unit.cpp

```diff
@@ -307,6 +307,8 @@ void CUnit::PreInit(const UnitLoadParams& params)
 
 	harvestStorage = unitDef->harvestStorage;
 
+	selectionRadius = unitDef->selectionRadius;
```

Comment:
Did you copy this as a separate variable, so it could be changed on a per-unit basis?
Otherwise `u->unitDef->selectionRadius` might work equally well in places you now use `u->selectionRadius`.
Not a big deal, just curious.

Reply:
I think I remember seeing some gadget that would scale unit sizes, maybe from sprung? Although I do not know how well supported that is with other engine things like collision volumes.